src/woo_publications/metadata/keep_organisations_values.py

```diff
@@ -0,0 +1,37 @@
+from contextlib import contextmanager
+
+from django.db import models, transaction
+
+from .models import Organisation
+
+
+@contextmanager
+@transaction.atomic
+def keep_organisations_values():
```

Comment:
I decided to not make this dynamic because doing that makes it quite hard to filter the Organisation model. Either we take them all (which I don't want to opt for) or we try to pry out the correct field filters based on the Organisation._meta.fields values (default, blank, null) but I don't know how good of a strategy that is.